.env

```diff
@@ -0,0 +1,2 @@
+FAASM_VERSION=0.5.11
```

Comment:
We only need `.env` for docker-compose, but I don't think we have any docker-compose stuff in here do we? Can we lose this? Any time we need `FAASM_VERSION` we can `cat` the `VERSION` in the faasm submodule